readme.md

```diff
@@ -1,14 +1,14 @@
-# filename-reserved-regex [![Build Status](https://travis-ci.org/sindresorhus/filename-reserved-regex.svg?branch=master)](https://travis-ci.org/sindresorhus/filename-reserved-regex)
+# filename-reserved-regex [![Build Status](https://travis-ci.com/sindresorhus/filename-reserved-regex.svg?branch=master)](https://travis-ci.com/sindresorhus/filename-reserved-regex)
 
 > Regular expression for matching reserved filename characters
 
-On Unix-like systems `/` is reserved and [`<>:"/\|?*`](http://msdn.microsoft.com/en-us/library/aa365247%28VS.85%29#naming_conventions) as well as non-printable characters `\x00-\x1F` on Windows.
+On Unix-like systems `/` is reserved and [`<>:"/\|?*`](https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#naming-conventions) as well as non-printable characters `\u0000-\u001F` on Windows.
```

Comment:
Due to [unicorn/no-hex-escape](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-hex-escape.md), I’ve updated to Unicode escapes in `index.js` and `test.js`. Therefore, I think it makes sense to update the readme as well.